src/posts/zkalc.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'zkalc: a cryptographic calculator'
+description: 'zkalc: a cryptographic calculator!'
+author: 'George Kadianakis, Michele Orrù'
+date: '2023-01-16'
+---
+
+<center>
```

Comment:
[Images are automatically centered on the posts](https://github.com/ethereum/cryptography-research-website#how-to-add-images-to-a-local-post), no need for the `<center>` tags